lidar_topic: "/cloud_filter/filtered_clouds"
yolo_topic: "/yolov5/BoundingBoxes"

extended_x_min: 0
extended_x_max: 0
extended_y_min: 0
extended_y_max: 0